encoder/vaapiencoder_vp9.cpp

```diff
@@ -257,10 +257,10 @@ bool VaapiEncoderVP9::fill(VAEncPictureParameterBufferVP9* picParam,
         }
     }
 
-    picParam->frame_width_src = width();
-    picParam->frame_height_src = height();
-    picParam->frame_width_dst = width();
-    picParam->frame_height_dst = height();
+    picParam->frame_width_src = ALIGN8(width());
```

Comment:
yes, there's going to be a green horizontal line on the  bottom of the encoded stream for resolution 2048x858 or a green horizontal line on the bottom and a green vertical line on the right of the encoded stream when resolution is for example 198x150.